src/lib.rs

```diff
@@ -337,12 +345,8 @@ impl Database {
 
             self.load_fonts_dir("/usr/share/fonts/");
             self.load_fonts_dir("/usr/local/share/fonts/");
-
-            if let Ok(ref home) = std::env::var("HOME") {
-                let home_path = std::path::Path::new(home);
-                self.load_fonts_dir(home_path.join(".fonts"));
-                self.load_fonts_dir(home_path.join(".local/share/fonts"));
-            }
+            self.load_fonts_dir(home_path.join(".fonts"));
```

Comment:
I don't think this part should be changed. And you're potentially passing `~` here, which would have to be expanded.